File: 1470. Shuffle the Array/solution.cpp
class Solution {
public:
  vector<int> shuffle(vector<int>& nums, int n) {
    vector<int> result;
    for (int index = 0; index < n; index++){
      result.push_back(nums[index]);
      result.push_back(nums[n + index]);
    }
    
    return result;
  }
};

File: 1470. Shuffle the Array/solution.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
  let result = [];
  for (let index = 0; index < n; index++){
    result.push(nums[index]);
    result.push(nums[n + index]);
  }
  
  return result;
};